src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -386,7 +386,7 @@ def upload_test_file(
     )
     logger.debug("Upload test file response: %s", response)
     logger.info("Upload test file completed")
-    return response
+    return response.as_dict()
```

Comment:
I believe so, I am seeing similar experience as the last version (1.0.1)
![image](https://github.com/user-attachments/assets/0e0230a9-54a1-4512-ac85-33d2fe900764)

I was expecting the patch method for upload file to return JSON https://github.com/Azure/azure-cli-extensions/blob/main/src/load/azext_load/vendored_sdks/loadtesting/_operations/_patch.py#L115, but since I'm needing to add the as_dict() here, looks like I'm not getting JSON but the model